augur/utils.py

```diff
@@ -25,18 +27,6 @@ class AugurException(Exception):
 
 
 @contextmanager
```

Comment:
We need to delete this line, too, so we don't turn `is_vcf` into a context manager.